commit b389b1806babbbf54cf92fe4c34c855a070a3cf4
Author: mjw <mjw@a5019735-40e9-0310-863c-91ae7b9d1cf9>
Date:   Wed Apr 22 15:29:03 2015 +0000

    Enable rt_sigpending syscall on ppc64 linux.
    
    Patch from Thomas Huth <thuth@redhat.com>
    
    git-svn-id: svn://svn.valgrind.org/valgrind/trunk@15133 a5019735-40e9-0310-863c-91ae7b9d1cf9

diff --git a/coregrind/m_syswrap/syswrap-ppc64-linux.c b/coregrind/m_syswrap/syswrap-ppc64-linux.c
index ba4116d..6f3c1f7 100644
--- a/coregrind/m_syswrap/syswrap-ppc64-linux.c
+++ b/coregrind/m_syswrap/syswrap-ppc64-linux.c
@@ -988,7 +988,7 @@ static SyscallTableEntry syscall_table[] = {
    LINXY(__NR_rt_sigaction,      sys_rt_sigaction),       // 173
    LINXY(__NR_rt_sigprocmask,    sys_rt_sigprocmask),     // 174
 
-// _____(__NR_rt_sigpending,     sys_rt_sigpending),      // 175
+   LINXY(__NR_rt_sigpending,     sys_rt_sigpending),      // 175
    LINXY(__NR_rt_sigtimedwait,   sys_rt_sigtimedwait),    // 176
    LINXY(__NR_rt_sigqueueinfo,   sys_rt_sigqueueinfo),    // 177
    LINX_(__NR_rt_sigsuspend,     sys_rt_sigsuspend),      // 178
